File: 1-Iniciante/c/1020_idade-em-dias.c
#include <stdio.h>

int main() 
{
    int ano = 0, mes = 0, dia = 0, A;
    scanf("%d", &A);
    while (A > 0) {
        if (A >= 365) {
            A -= 365;
            ano++; 
        } else if (A >= 30) {
            A -= 30;
            mes++;
        } else {
            A -= 1;
            dia++;
        }
    } 
    printf("%d ano(s)\n", ano);
    printf("%d mes(es)\n", mes);
    printf("%d dia(s)\n", dia);
    return 0;
}

File: 1-Iniciante/python/1020_idade-em-dias.py
dias = {'ano': 0, 'mes': 0, 'dia': 0}
A = int(input())
while A != 0:
    if A >= 365:
        A -= 365
        dias['ano'] += 1
    elif A >= 30:
        A -= 30
        dias['mes'] += 1
    else:
        A -= 1
        dias['dia'] += 1
print(f"{dias['ano']} ano(s)\n{dias['mes']} mes(es)\n{dias['dia']} dia(s)")